src/main/proto/spine/options.proto

```diff
@@ -1299,31 +1299,59 @@ message IfHasDuplicatesOption {
 //
 // For unbounded ranges, please use `(min)` and `(max) options.
 //
+// The option supports all Protobuf numeric fields.
+//
 message RangeOption {
 
     // The default error message.
     option (default_message) = "The field `${parent.type}.${field.path}` must be within"
-        " the following range: `${range.value}`.";
+        " the following range: `${range.value}`. The passed value: `${field.value}`.";
 
     // The string representation of the range.
     //
-    // The range can be open (not including the endpoint) or closed (including the endpoint) on
-    // each side. Open endpoints are indicated using a parenthesis (`(`, `)`). Closed endpoints are
-    // indicated using a square bracket (`[`, `]`).
+    // ## Syntax
+    //
+    // 1. The range can be open (not including the endpoint) or closed (including the endpoint)
+    //    on each side. Open endpoints are indicated using a parenthesis (`(`, `)`).
+    //    Closed endpoints are indicated using a square bracket (`[`, `]`).
+    // 2. The lower and upper endpoints are separated by either the `..` or ` .. ` delimiter.
+    // 3. The lower bound must be strictly less than the upper bound.
+    //
+    // ## Field type consistency
+    //
+    // The definition of the range must be consistent with the field type it describes.
     //
-    // Example: Defining ranges of numeric values.
+    // 1. A range for an integer field must use integer numbers. Specifying an integer
+    //    with a decimal separator (`.`) is not allowed, even if the value has no
+    //    fractional part (e.g., `5.0`).
     //
-    //     message NumRanges {
+    //    Example: Defining ranges for integer fields.
+    //
+    //    message IntegerRanges {
     //         int32 hour = 1 [(range).value = "[0..24)"];
     //         uint32 minute = 2 [(range).value = "[0..59]"];
-    //         float degree = 3 [(range).value = "[0.0..360.0)"];
-    //         double angle = 4 [(range).value = "(0.0..180.0)"];
     //     }
     //
-    // NOTE: That definition of ranges must be consistent with the type they describe.
-    //       An range for an integer field must be defined with integer endpoints.
-    //       A range for a floating point field must be defined with decimal separator (`.`),
-    //       even if the endpoint value does not have a fractional part.
+    // 2. A range for a floating point field must always use a decimal separator (`.`),
+    //    even if the value value has no fractional part. An exponent part
+    //    (`E` or `e`, followed by an optional sign and digits) is allowed.
```

Comment:
As long as you mention it, you should have an example of such a value, in this point, not in the next one.